lib/rules/declaration-property-value-no-unknown/index.mjs

```diff
@@ -71,6 +71,8 @@ const rule = (primary, secondaryOptions) => {
 			'text-spacing-trim': 'normal | space-all | space-first | trim-start',
 			'text-wrap-style': 'auto | balance | pretty | stable',
 			'word-break': '| auto-phrase',
+			'text-wrap': 'wrap | nowrap | balance | pretty | stable',
+			'text-wrap-mode': 'wrap | nowrap',
```

Comment:
```suggestion
			'text-wrap-mode': 'wrap | nowrap',
			'text-wrap': "<'text-wrap-mode'> || <'text-wrap-style'>",
```

Reply:
Just confirming:
	`'text-wrap': "<'text-wrap-mode'> || <'text-wrap-style'>",`

wrapping under this `<>` will be recognised while parsing the `propertiesSyntax`?

Reply:
You can try to add a test if you don't believe me :)